### PR TITLE
packaging: merge debian packaging changes back into master

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: packaging/debian-sid/patches/0016-skip-TestPopulateFromSeedWithConnectHook.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/overlord/devicestate/firstboot_test.go b/overlord/devicestate/firstboot_test.go
+index 47ac7a7a90..19b505673b 100644
+--- a/overlord/devicestate/firstboot_test.go
++++ b/overlord/devicestate/firstboot_test.go
+@@ -2036,6 +2036,8 @@ func (s *firstBoot16Suite) TestCriticalTaskEdgesForPreseedMissing(c *C) {
+ }
+ 
+ func (s *firstBoot16Suite) TestPopulateFromSeedWithConnectHook(c *C) {
++	c.Skip("not converging on debian")
```
Comment (zyga) [inline packaging/debian-sid/patches/0016-skip-TestPopulateFromSeedWithConnectHook.patch:9]: The reason this test is failing on Debian is missing mocking and the Debian network sandbox. I've fixed it here https://github.com/snapcore/snapd/pull/12385 - you can drop this patch once this lands.

**Comment by zyga** [comment]
@mvo5 apologies for switching this from draft mode, I meant my other PR and got confused with lots of tabs open.

File: packaging/debian-sid/patches/0016-skip-TestPopulateFromSeedWithConnectHook.patch (hunk 1 above)
Comment (mvo5) [reply packaging/debian-sid/patches/0016-skip-TestPopulateFromSeedWithConnectHook.patch:9]: Thanks, added a comment

**Comment by Meulengracht** [comment]
This PR has conflicts

**Review by MiguelPires — APPROVED** [review]
LGTM. @mvo5 this has a conflict so it won't build

**Pull request merged**